clients/admin-ui/src/features/poc/privacy-notices-sandbox/components/ExperienceConfigSection.tsx

```diff
@@ -103,9 +103,9 @@ const ExperienceConfigSection = ({
             headerColor={palette.FIDESUI_MINOS}
             emptyMessage="Available notices will appear here after fetching experience"
           >
-            {privacyNotices.length > 0 && (
+            {privacyNotices.length > 0 ? (
               <PrivacyNoticesList privacyNotices={privacyNotices} />
-            )}
+            ) : null}
```

Comment:
is there a reason this got changed? We generally prefer the former battern for rendering components